rocAL/rocAL/source/rali_api_meta_data.cpp

```diff
@@ -422,3 +430,52 @@ RALI_API_CALL raliCopyEncodedBoxesAndLables(RaliContext p_context, float* boxes_
         memcpy(temp_bbox_buf, meta_data.second->get_bb_cords_batch()[i].data(), sizeof(BoundingBoxCord) * bb_count);
     }
 }
+
+void
+RALI_API_CALL raliGetJointsDataPtr(RaliContext p_context, RaliJointsData **joints_data)
+{  
+    if (!p_context)
+        THROW("Invalid rali context passed to raliGetBoundingBoxCords")
+    auto context = static_cast<Context*>(p_context);
+    auto meta_data = context->master_graph->meta_data();
+    size_t meta_data_batch_size = meta_data.second->get_joints_data_batch().center_batch.size();
+
+    if(context->user_batch_size() != meta_data_batch_size)
+        THROW("meta data batch size is wrong " + TOSTR(meta_data_batch_size) + " != "+ TOSTR(context->user_batch_size() ))
+    if(!meta_data.second)
+    {
+        WRN("No label has been loaded for this output image")
+        return;
+    }
+
+    *joints_data = (RaliJointsData *)(&(meta_data.second->get_joints_data_batch()));
+}
+
+
+void
+RALI_API_CALL raliGetJointsData(RaliContext p_context, RaliJointsData *joints_data)
+{  
+    if (!p_context)
+        THROW("Invalid rali context passed to raliGetBoundingBoxCords")
+    auto context = static_cast<Context*>(p_context);
+    auto meta_data = context->master_graph->meta_data();
+    size_t meta_data_batch_size = meta_data.second->get_joints_data_batch().image_id_batch.size();
+
+    if(context->user_batch_size() != meta_data_batch_size)
+        THROW("meta data batch size is wrong " + TOSTR(meta_data_batch_size) + " != "+ TOSTR(context->user_batch_size() ))
+    if(!meta_data.second)
+    {
+        WRN("No label has been loaded for this output image")
+        return;
+    }
+
+    joints_data->image_id_batch = meta_data.second->get_joints_data_batch().image_id_batch;
+    joints_data->annotation_id_batch = meta_data.second->get_joints_data_batch().annotation_id_batch;
+    joints_data->image_path_batch = meta_data.second->get_joints_data_batch().image_path_batch;
+    joints_data->center_batch = meta_data.second->get_joints_data_batch().center_batch;
+    joints_data->scale_batch = meta_data.second->get_joints_data_batch().scale_batch;
+    joints_data->joints_batch = meta_data.second->get_joints_data_batch().joints_batch;
+    joints_data->joints_visibility_batch = meta_data.second->get_joints_data_batch().joints_visibility_batch;
+    joints_data->score_batch = meta_data.second->get_joints_data_batch().score_batch;
+    joints_data->rotation_batch = meta_data.second->get_joints_data_batch().rotation_batch;
+}
```

Comment:
Why do we have 2 calls? raliGetJointsData & raliGetJointsDataPtr?

Reply:
Removed the aliGetJointsData api since it is not needed